code/datums/components/secbot_command.dm

```diff
@@ -0,0 +1,31 @@
+/datum/component/secbot_command
+	dupe_mode = COMPONENT_DUPE_HIGHLANDER
```

Comment:
last wins means if you have a whistle in hand you are practically immune to securitrons, right? it’ll be detected as contra, you whistle, they stop?

Reply:
Yep, but you have to keep whistling at them which I think is a fair tradeoff